nilearn/plotting/surface/surf_plotting.py

```diff
@@ -411,7 +414,6 @@ def plot_surf_stat_map(
     view=None,
     engine=DEFAULT_ENGINE,
     cmap=DEFAULT_DIVERGING_CMAP,
-    symmetric_cmap=None,
```

Comment:
@Remi-Gau Is there a specific reason to introduce `symmetric_cbar` here but not in `plot_surf`?

Reply:
nope

Reply:
but modifying that would mean applying this codeblock to plot_surf

```
    cbar_vmin, cbar_vmax, vmin, vmax = (
        backend._adjust_colorbar_and_data_ranges(
            loaded_stat_map,
            vmin=vmin,
            vmax=vmax,
            symmetric_cbar=symmetric_cbar,
        )
    )
```

Reply:
Yes. But I am not sure if it is necessary. 

My questions are: 

- Why is it necessary in one function but not the other?

- Are there cases where one wants the cbar to be symmetric, but not cbar color, or vice versa?